docs/cn/RocketMQ_Example.md

```diff
@@ -438,6 +438,7 @@ public class ScheduledMessageConsumer {
    public static void main(String[] args) throws Exception {
       // 实例化消费者
       DefaultMQPushConsumer consumer = new DefaultMQPushConsumer("ExampleConsumer");
+      consumer.setNamesrvAddr("127.0.0.1:9876");
```

Comment:
NamesrvAddr is not necessary for the sample code, you can look at the official website sample （ https://rocketmq.apache.org/docs/quick-start/ ）, export NAMESRV_ADDR=localhost:9876